tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<p>Write more.</p><ul><li>a, b should be the same dtype, device</li><li class="ql-indent-1">what happen if they are different?</li><li>what are the ouptut dtype of such operations?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>